packages/mobile/src/send/SendAmount.tsx

```diff
@@ -353,14 +353,16 @@ const styles = StyleSheet.create({
   },
   contentContainer: {
     flex: 1,
+    width: Dimensions.get("window").width,
   },
   showAmountContainer: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'center',
+   flex: 1,
+   flexDirection: 'row',
+   justifyContent: 'center',
```

Comment:
Looks like this was manually formatted. We should let prettier do this.
There's a default configuration with vscode that does it on save: https://github.com/celo-org/celo-monorepo/blob/fce3df17d1df43c37e59a3cc6918cc1a788a7dad/.vscode/settings.json#L38
The vscose prettier plugin needs to be installed too.

Reply:
ahh! i have prettier installed but it doesnt always format on save. might just reinstall vscode since its been buggy